examples/todoapp/lib/base/models/todo_model.dart

```diff
@@ -7,8 +7,8 @@
 
 import 'package:equatable/equatable.dart';
 import 'package:json_annotation/json_annotation.dart';
-import 'package:rx_bloc_list/models.dart';
-
+// ignore: implementation_imports
+import 'package:rx_bloc_list/src/models/identifiable.dart';
```

Comment:
Why we need to import a file instead of a library ?

Reply:
The issue is that rx_bloc_list `PaginatedList` imports flutter/cupertino library, and when sharing the model with the server it doesn't compile

Reply:
Please create an issue for that. The models are part of the business layer and should be UI independent